components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -283,6 +287,38 @@ func parseLocationSpecFromCLIArgs(location string) (backup.LocationSpecification
 			SecretKey:    backupCmdFlags.s3SecretKey,
 			SessionToken: backupCmdFlags.s3SessionToken,
 		}, nil
+	} else if strings.HasPrefix(location, "gcs://") {
+		bucketAndBasePath := strings.TrimPrefix(location, "gcs://")
+		parts := strings.SplitN(bucketAndBasePath, "/", 2)
+		bucketName := ""
+		basePath := ""
+		if len(parts) == 1 {
+			bucketName = parts[0]
+		} else if len(parts) == 2 {
+			bucketName = parts[0]
+			basePath = parts[1]
+		} else {
+			return nil, status.Errorf(status.InvalidCommandArgsError,
+				"%q could not be parsed. The expected input is gcs://bucket/base/path",
+				location,
+			)
+		}
+
+		creds := ""
+		if backupCmdFlags.gcsCredentialsPath != "" {
```

Comment:
it might be good to explicitly require this be set for the restore case

Reply:
How are you passing the json in the tests? Are we going to expect customers to paste the whole string to the cli or a pointer to a file containing it?

Reply:
gcsCredentialsPath would be a path to the json file. When using the cli, you would set this or use the environment variable that also points to a path. It gets read in below